leaflet/templates/leaflet/widget.html

```diff
@@ -41,5 +41,11 @@
 {% endblock map %}
 {% endif %}
 
-{% if display_raw %}<p> Geometry:</p>{% endif %}
-<textarea id="{{ id_css }}" class="required" cols="150" rows="10" name="{{ name }}">{{ serialized }}</textarea>
+{% if display_raw %}
+<p>Draw on map or paste a <a
+    href="http://geojsonlint.com/"
+    title="Validate your GeoJSON Geometry at GeoJSONLint.com in a new window"
+    target="_"
```

Comment:
Should be `_blank`

Reply:
Fixed by https://github.com/makinacorpus/django-leaflet/pull/249/commits/a23f6aa42735325973ff47d3c3f1134461bedd00

Reply:
Changes added in https://github.com/makinacorpus/django-leaflet/pull/249/commits/68b7f4701d0440b9d43b10d158136a7a5de8963f